tests/phpunit/tests/import/import.php

```diff
@@ -6,6 +6,12 @@
  * @group import
  */
 class Tests_Import_Import extends WP_Import_UnitTestCase {
+
+	protected $expected_deprecated = array(
+		'wp_slash_strings_only',
+		'addslashes_strings_only',
+	);
```

Comment:
@jrfnl, @hellofromtonya All but one of the test methods in this class will expect deprecation notices for these functions.

By defining them here, it means that `$this->expected_deprecated = array()` can be called in the one test that doesn't expect these deprecation notices to occur, and doesn't apply to later tests in the class.

While this makes sense, I haven't seen this approach used in Core's test suite before, so I'm pinging for your thoughts. Note: This is also done in another file in this PR.

Reply:
Just my two pennies:

I'm not keen on this as it obfuscates what's going on and effectively "hides" that the deprecation is being expected and tested for every single function (save one).

While it works, it requires a deep understanding of the test suite framework to understand what's going on, which makes these tests error prone going forward.

Reply:
As a frustrated side-note (not to be actioned in this PR): _why the heck is this even possible and why is the `$expected_deprecated` property in the `WP_UnitTestCase_Base` class `protected instead of `private` ???_